src/Power.cpp

```diff
@@ -532,6 +591,10 @@ bool Power::analogInit()
 
     batteryLevel = &analogLevel;
     return true;
+#elif HAS_TELEMETRY && !MESHTASTIC_EXCLUDE_ENVIRONMENTAL_SENSOR && !defined(ARCH_PORTDUINO)
+    batteryLevel = &analogLevel;
+    return true; // TODO ABK: should be hasINA()
+
```

Comment:
Definitely gotta sort this TODO first

Reply:
> Definitely gotta sort this TODO first

got this addressed now. Was way easier than I thought at first glance.